draft-ietf-tls-ticketrequests.md

```diff
@@ -47,32 +47,33 @@ normative:
 --- abstract
 
 TLS session tickets enable stateless connection resumption for clients without
-server-side per-client state. Servers vend session tickets to clients, at their
-discretion, upon connection establishment. Clients store and use tickets when
-resuming future connections. Moreover, clients should use tickets at most once for
-session resumption, especially if such keying material protects early application
-data. Single-use tickets bound the number of parallel connections a client
-may initiate by the number of tickets received from a given server. To address
-this limitation, this document describes a mechanism by which clients may specify
-the desired number of tickets needed for future connections.
+server-side, per-client state. Servers vend an arbitrary number of session tickets
+to clients, at their discretion, upon connection establishment. Clients store and
+use tickets when resuming future connections. This document describes a mechanism by
+which clients may specify the desired number of tickets needed for future connections.
+This extension aims to remove server ambiguity surrounding ticket count while simultaneously
```

Comment:
This early in the document, it's a bit unclear what "server ambiguity surrounding ticket count" means.

Reply:
Hmm, I'm not sure adding more text is appropriate here. Do you have an alternative way to phrase this?

Reply:
Yeah, I agree we don't want more text, but a clearer statement would be nice. How about:
`This extension aims to provide a means for servers to determine the number of tickets to generate, while simultaneously...`

Reply:
I also like it :) I went with another variant: "This extension aims to provide a means for servers to determine the number of tickets to generate as a means of reducing ticket waste, while simultaneously priming clients for future connection attempts."